src/renderer/analytics/segment.js

```diff
@@ -40,7 +41,7 @@ const extraProperties = store => {
   const state: State = store.getState();
   const { language, region } = langAndRegionSelector(state);
   const systemLocale = getSystemLocale();
-  const device = getCurrentDevice(state);
+  const device = getCurrentDevice(state) || lastSeenDeviceSelector(state);
   const deviceInfo = device ? { modelId: device.modelId } : {};
```

Comment:
I see two issues in that:

- getCurrentDevice returns a Device but lastSeenDeviceSelector returns a DeviceModelInfo
- it doesn't solve LL-3900: we want to add in the "deviceInfo" object that I think comes from DeviceModelInfo

My proposal is to actually drop the "getCurrentDevice" usage. in the lastSeenDeviceSelector, we have enough we need and we can just use that. 

https://github.com/LedgerHQ/ledger-live-common/blob/5fd037de618d245e960046e69aab240c847cc7d2/src/types/manager.js#L31-L35

i would suggest that we track `{ modelId, deviceVersion, appLength }` (deviceVersion is deviceInfo.version, appLength is apps.length )

kinda consistently with https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/renderer/screens/manager/Dashboard.js#L73-L76